components/table/ProductTable.tsx

```diff
@@ -32,10 +41,19 @@ const ProductTable = () => {
       </div>
       <div className='table-body'>
         <ProductTableHeader />
-        <div className='product-table-row-section'>
-          {productsList.map((product, key) => (
-            <ProductTableRow product={product} key={`product-${key}`} />
-          ))}
+        <div id='scrollableDiv' className='product-table-row-section'>
+          <InfiniteScroll
+            scrollableTarget='scrollableDiv'
+            dataLength={40}
+            next={handleLoadMoreData}
+            hasMore={hasMoreData}
+            loader={<p className='data-loader-message'>Loading more data...</p>}
```

Comment:
Can we put loader in separate component and reuse it across different parts of application? I'm not sure if at the moment that's only place where it's used, but in future we definitely have more scrolls and paginations, it be good to have consistent loading option. Also, I think `Loading more data...` should be translated.

Reply:
Good point!